src/snowflake/snowpark/dataframe.py

```diff
@@ -165,10 +170,26 @@ def _get_unaliased(col_name: str) -> List[str]:
     return unaliased
 
 
-def _alias_if_needed(df: "DataFrame", c: str, prefix: str, common_col_names: List[str]):
+def _alias_if_needed(
+    df: "DataFrame",
+    c: str,
+    prefix: Optional[str],
+    suffix: Optional[str],
+    common_col_names: List[str],
+):
     col = df.col(c)
     unquoted = c.strip('"')
```

Comment:
Let's rename this to `unquoted_col_name` given now there are multiple strings that can be quoted/unquoted?

Reply:
Done